HHS/Head-Start-TTADP
# [TTAHUB-2399] Remove feature flags

## Description of change

Remove feature flags: resources_dashboard, merge_goals, and monitoring.

Also solves this issue:
![visual bug](https://github.com/HHS/Head-Start-TTADP/assets/3288586/edb59235-9166-4d55-86cd-cff86eae7990)


## How to test

To test the feature flag removal, run the migration and make sure that the appropriate users can see the content specified (the biggest risk here is that I missed a FeatureFlag component somewhere on the frontend)

To test the CSS fix, recreate the filter config pictured above and confirm there is no overflow

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2399


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
